src/components/DefaultContainer/DefaultContainer.tsx

```diff
@@ -70,40 +84,117 @@ export const DefaultContainer = () => {
     setIsVisibleAddMailModal((prevState: boolean) => !prevState);
   };
 
-  const navigateToDashboard = (props: RouteComponentProps) => (
-    <Dashboard
-      {...props}
-      onGetUserProfile={handleGetUserProfile}
-      spidMail={userProfile.email}
-      onWorkMailSet={handleWorkMailSet}
-      toggleAddMailModal={toggleAddMailModal}
+  /**
+   * Given a cookie key `cookieName`, returns the value of
+   * the cookie or empty string, if the key is not found.
+   */
+  function getCookie(cookieName: string): string {
```

Comment:
please move this function in a utility package or - better - use some library like https://www.npmjs.com/package/react-cookie